main.tf

```diff
@@ -36,16 +58,17 @@ resource "aws_launch_configuration" "cfg" {
                 EOF
 }
 
-resource "aws_autoscaling_group" "asg" {
-  name = "premium_asg"
+resource "aws_autoscaling_group" "asg_group" {
```

Comment:
Rename resource `aws_autoscaling_group.nginx`

Reply:
Renamed resource from aws_autoscaling_group.asg_group to  aws_autoscaling_group.nginx